internal/auth0/organization/data_source.go

```diff
@@ -129,10 +129,10 @@ func fetchAllOrganizationConnections(ctx context.Context, api *management.Manage
 
 func fetchAllOrganizationMembers(ctx context.Context, api *management.Management, organizationID string) ([]string, error) {
```

Comment:
Should this return an `[]management.OrganizationMember` and let the flatten funcs convert the necessary fields (user_id into slice of strings) ? Otherwise it seems we're deviating from the convention inside the provider just for this use case.